packages/playground/tsconfig-json/__tests__/tsconfig-json.spec.ts

```diff
@@ -48,7 +46,22 @@ describe('transformWithEsbuild', () => {
     })
     // "importsNotUsedAsValues": "preserve" from tsconfig.json should not be read
     // and defaults to "remove"
-    expect(result.code).not.toContain(
+    expect(result.code).not.toContain('import "./not-used-type";')
+  })
+
+  test.skip('preserveValueImports', async () => {
+    const main = path.resolve(__dirname, '../src/main.ts')
+    const mainContent = fs.readFileSync(main, 'utf-8')
+    const result = await transformWithEsbuild(mainContent, main, {
+      tsconfigRaw: {
+        compilerOptions: {
+          useDefineForClassFields: false
+          // preserveValueImports: true // TODO: preserveValueImports isn't in the types definition in esbuild yet
```

Comment:
should we file an issue about this? or is there an existing one we can link to from here?

Reply:
I was waiting to see if someone could confirm this is an issue or ai missed something. Ill file an issue if not 👍

Reply:
I don't see it. I just filed an issue here: https://github.com/evanw/esbuild/issues/1816

Reply:
@benmccann 0.14.3 fixes this I think.

Reply:
https://github.com/evanw/esbuild/commit/56da27e3d5ab590dd15412bddb58925ea9941363